### PR TITLE
added unit tests for api batch call

### PR DESCRIPTION
**Pull request by dehang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # `edl-cta-sdk`
 
 <!-- AUTO-GENERATED-CONTENT:START (VERSION_BADGE:link=false) -->
-![npm](https://img.shields.io/badge/npm-v0.0.5-green.svg)
+![npm](https://img.shields.io/badge/npm-v0.0.0-green.svg)
```
Comment (setheen) [inline README.md]: Not sure this version reflects version in package.json.  Run `yarn markdown` first?

File: src/get-driver-summary.ts [hunk 2]
```diff
@@ -44,5 +44,5 @@ export function getDriverSummaryBatch(
       log.error(err);
       throw new Error(err);
     })
-    .then(response => response as IDriverSummaryResponse[]); // TODO: couldn't get this to return above
+    .then(response => response as IDriverSummaryResponse[]);
```
Comment (setheen) [inline src/get-driver-summary.ts]: Is this needed to satisfy a typing error?  The code doesn't actually do anything besides cast so I wonder if you can just remove line 47 altogether?

File: README.md (hunk 1 above)
Comment (DSchau) [reply README.md]: @setheen it does :) 

https://github.com/PeopleNet/ise-payroll-sdk/blob/master/package.json#L3

File: src/get-driver-summary.ts (hunk 2 above)
Comment (DSchau) [reply src/get-driver-summary.ts]: response _should_ be any, so it'd be able to be typed without the explicit cast. I think the normalization is what we'd want here though, so I think it's more of a TBD thing

Comment (dehang) [reply src/get-driver-summary.ts]: As in we could normalize the response data to some PFM data representation? If that's the case, then yea, we can leave this as is for TBD.

File: README.md (hunk 1 above)
Comment (dehang) [reply README.md]: so what should this be? it needs to match the sdk version? 0.0.1

**Pull request merged**